source/_integrations/eway.markdown

```
@@ -0,0 +1,170 @@
+{
+  "netFirmVer": 1.2,
+  "appFirmVer": 2.1,
+  "wifiSsid": "MyWiFi",
+  "ip": "192.168.1.100",
+  "wifiIsNormal": 0,
+  "isLock": 0,
+  "board": []
+}
+```
+
+## Troubleshooting
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)